resources/api/kio-api.yaml

```diff
@@ -388,7 +388,7 @@ parameters:
     type: string
     description: ID of the version
     required: true
-    pattern: "^[a-zA-Z0-9][a-zA-Z0-9.-]*[a-zA-Z0-9]$"
+    pattern: "^[A-Za-z0-9]([A-Za-z0-9\._-]*[A-Za-z0-9])?$"
```

Comment:
I would use a non-capturing group here. "\" is not needed to match the "." literally (within "[ ... ]" brackets). My suggestion for the regex: `^[A-Za-z0-9](?:[A-Za-z0-9._-]*[A-Za-z0-9])?$`